provisioning_templates/finish/preseed_default_finish.erb

```diff
@@ -29,6 +29,7 @@ oses:
 /bin/hostname  <%= @host.shortname %>.<%= @host.domain %>
 <% end -%>
 
+<%= snippet_if_exists(template_name + " custom snippet") %>
```

Comment:
I wasn't sure if I should put anything in `provision/preseed_default.erb` or where it would best go. (I'm only building EL hosts and am not familiar with preseed)

Reply:
This is fine, I'd probably name this " custom script".